Corzbank.Web/ClientApp/src/app/header/wallet/wallet.component.html

```diff
@@ -136,52 +225,41 @@ <h2>Transfer to Card</h2>
 			<form [formGroup]="transferForm">
 				<div class="transfer">
 					<div class="field">
-						<input
-							formControlName="receiverCardNumber"
-							type="text"
-							maxlength="28"
-							placeholder="**** **** **** ****"
-							maxlength="16"
-							class="form-control"
-						/>
-						<small
-							*ngIf="
+						<input formControlName="receiverCardNumber" type="text" maxlength="28"
+							placeholder="**** **** **** ****" maxlength="16" class="form-control" />
```

Comment:
You have 2 same attributes - maxlength. One has value 28, another - 16

Reply:
resolved